klasses/views.py

```diff
@@ -72,3 +79,35 @@ def get(self, request, username, *args, **kwargs):  # pylint: disable=unused-arg
         )
         klass = self.get_object()
         return Response(serialize_user_klass(user=user, klass=klass))
+
+
+class UserKlassStatement(RetrieveAPIView):
```

Comment:
I'm using the `lookup_field` et al to automatically pull a `Klass` that matches the provided `klass_key` (or a 404 if it doesn't exist). The view above this is doing the same. I think that might be the only piece of functionality that we're getting here beyond a vanilla Django view. Any specific concerns about using DRF here?